src/integrate/orchestrator.rs

```diff
@@ -552,14 +552,14 @@ ignore_txs = false
             entrypoint: Some(vec![]),
             host_config: Some(HostConfig {
                 port_bindings: Some(port_bindings),
-
                 binds: Some(vec![
                     format!("{}/conf:/etc/bitcoin", devnet_config.working_dir),
                     format!(
                         "{}/data/{}/bitcoin:/root/.bitcoin",
                         devnet_config.working_dir, boot_index
                     ),
                 ]),
+                extra_hosts: Some(vec!["host.docker.internal:172.17.0.1".into()]),
```

Comment:
Last time I looked at this issue (over a year ago), I think it was possible for the docker gateway to be a different IP. Maybe needs to be detected, or configurable?

Reply:
it is possible, that was my thought as well - hardcoding the IP here will surface issues with users that use different subnets for docker

Reply:
Yeah the hard coded IP address was an unsuccessful attempt to patch the issue, will revert.